blueprint/abac/office-365.md

```diff
@@ -696,11 +696,11 @@ The following mail flow rules have been configured for all implementation types.
 
 `Exchange Online Admin Centre > Mail flow > Rules`
 
-Note, the subject prepend rules listed here are not exhaustive. Adjust for each classification combination in use within the agency.
+#### Outgoing X-Protective-Marker (multiple rules)
 
-#### UNOFFICIAL - prepend subject
+Note 1, the `x-protective-marking` rules listed here are not exhaustive and are an example that can be used for each classification used within the Agency.
 
-Note, the PSPF recommends the append of the subject text rather than the prepend, at time of writing there is no transport rule for 'append'. If the Agency has the ability to transform mail messages using a gateway appliance then this is the recommended approach.
+Note 2, at time of writing the ability to add the `ORIGIN=user@agency.gov.au` parameter is not possible using a transport rule. If the Agency has the ability to transform mail messages using a gateway appliance then this is the current recommended approach for updating header values.
```

Comment:
What do you think about putting in dates when referring to "at time of writing", eg: `as of February 2022`

Reply:
@zedzedzed I agree, date (month/year) would handy. Will update reference

Reply:
@zedzedzed updated that reference. Note there is another one referring to MS Forms which will be updated separate PR.